### ISSUE TITLE
Lazy load should be able to support aria-label and alt for images

### ISSUE THREAD

**Issue by ronilaukkarinen** [digitoimistodude/air-helper#27]
It should be able to add alt or aria-label for images. If not set manually, get text from WordPress media alternative text.

**Comment by ronilaukkarinen** [comment]
So,

By default there should be a way to add descriptive text manually. If not set, let's try to get alternative text from WP library image metadata, if such does not exist, we will have `alt=""` for images and `aria-hidden="true"` for divs.
